flepimop/gempyor_pkg/tests/seir/test_ic.py

```diff
@@ -44,7 +44,7 @@ def test_IC_allow_missing_node_compartments_success(self):
         sic.get_from_config(sim_id=100, modinf=s)
 
     def test_IC_IC_notImplemented_fail(self):
-        with pytest.raises(NotImplementedError, match=r".*unknown.*initial.*conditions.*"):
+        with pytest.raises(NotImplementedError, match=r"^Unknown initial conditions method \[received: .*?\]\.$"):
```

Comment:
This is both too prescriptive and not actually checking enough - because the received part is an open ended regex, doesn't matter what's there. We should be checking that the error message when offered XYZ for method replies that XYZ is garbage - that is, we should be checking for XYZ in the message.